src/main/java/com/google/research/bleth/services/DatabaseService.java

```diff
@@ -0,0 +1,158 @@
+package com.google.research.bleth.services;
+
+import com.google.appengine.api.datastore.*;
```

Comment:
Try to avoid wildcard imports.
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports